content/_partners/faq._en.md

```diff
@@ -73,59 +69,33 @@ logistics_accordion:
             An Interagency Agreement (IAA) is a type of contract. Login.gov is a cost-recoverable federal service, which means we must, by law, charge other agencies for our work. Our partnership and financial engagement will be governed by the IAA. 
     -
         id: b-2
-        title: What is the timeline for a new IAA?
+        title: Does Login.gov provide authorization?
         content: >-
-            Six to eight weeks. 
+            Login.gov does not provide authorization. At this time, Login.gov supports authentication and identity verification capabilities. We encourage agencies to take the lead on determining the best strategy for their role management and authorization. 
     -
         id: b-3
-        title: What is the timeline for a IAA modifications?
+        title: What is identity verification?
         content: >-
-            Six to eight weeks. 
+            “Identity proofing is the process by which a CSP (credentialing service provider) collects, validates, and verifies information about a person.” NIST SP 800-63-3, Digital Identity Guidelines This is the process Login.gov uses to verify that a user is who they say they are. While many agencies can validate an individual’s identity through an in-person proofing experience, we developed an online application that allows individuals to have their identities verified from their smartphone or computer.
```

Comment:
I think some stying needs to be added for the NIST reference at the beginning. Is it enough to just make the NIST SP reference italic, like in the content doc?

Reply:
Kinda sloppy-looking, if ya ask me!

Reply:
Ooh - yes. I was just blindly copy-pasting. Hopefully this looks a little better. I took license and added a `-` to indicate that it was a citation for the quote.

![Screen Shot 2022-08-05 at 1 05 45 PM](https://user-images.githubusercontent.com/93271895/183126580-b9667f19-aa30-4557-abe6-f92794e4cf25.png)

Reply:
Much better, thanks!